Fix link for doc instructions (#2607)
windows-toolkit/WindowsCommunityToolkit: .github/PULL_REQUEST_TEMPLATE.md

Please check if your PR fulfills the following requirements:

> - [ ] Tested code with current [supported SDKs](..#supported)

Reviewer: ```suggestion
- [ ] Tested code with current [supported SDKs](https://github.com/windows-toolkit/WindowsCommunityToolkit#supported)
```

> - [ ] Tested code with current [supported SDKs](../#-supported-sdks)

Reviewer 2: ```suggestion
- [ ] Tested code with current [supported SDKs](..#supported)
```

> - [ ] Pull Request has been submitted to the documentation repository [instructions](../blob/master/contributing.md#-adding-documentation). Link: <!-- docs PR link -->

Reviewer: ```suggestion
- [ ] Pull Request has been submitted to the documentation repository [instructions](https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs#contributing). Link: <!-- docs PR link -->
```

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../readme.md#supported)

> - [ ] Pull Request has been submitted to the documentation repository [instructions](../blob/master/contributing.md#-adding-documentation). Link: <!-- docs PR link -->

Reviewer 3: The other ones are also not working. We should add /blob/master/ to all of them ("supported SDKs" and "Thumbnail Style Guide and templates" links too).

Author: only the SDKs link was broken. The template style link works
